backend/src/schema/resolvers/login.spec.js

```diff
@@ -0,0 +1,80 @@
+const { userData } = require("../../data");
+
+const { gql } = require("apollo-server");
+const { createTestClient } = require("apollo-server-testing");
+
+const { constructTestServer } = require("../../utils/__utils");
+
+const POST_LOGIN = gql`
+  mutation login($username: String) {
+    login(username: $username) {
+      isLoggedIn
+      token
+    }
+  }
+`;
+
+const POST_SIGNUP = gql`
+  mutation signup($username: String) {
+    signup(username: $username) {
+      username
+      isLoggedIn
+      token
+    }
+  }
+`;
+
+let mutate;
+
+beforeAll(() => {
+  const { testServer } = constructTestServer();
+  mutate = createTestClient(testServer).mutate;
+});
+
+describe("Mutations", () => {
+  it("receiving token on login", async () => {
```

Comment:
```suggestion
  describe('given valid login credentials', () => {
    it("responds with token", async () => {
```
Always use simple present and make the `it` part of the sentence. Describe what `it` does.